docker-compose.yml

```diff
@@ -189,13 +188,22 @@ services:
       - "8964:8000"
 
   user-docs:
-    build:
-      context: .
-      args:
-        DEVEL: "yes"
+    image: warehouse:docker-compose
     command: mkdocs serve -a 0.0.0.0:8000 -f mkdocs-user-docs.yml
     volumes:
       - ./mkdocs-user-docs.yml:/opt/warehouse/src/mkdocs-user-docs.yml:z
       - ./docs/user:/opt/warehouse/src/docs/user:z
     ports:
       - "10000:8000"
+    depends_on: [web]
+
+  blog:
+    image: warehouse:docker-compose
+    command: mkdocs serve -a 0.0.0.0:8000 -f docs/blog.yml
+    volumes:
+      - ./docs/blog.yml:/opt/warehouse/src/docs/blog.yml:z
+      - ./docs/blog:/opt/warehouse/src/docs/blog:z
+      - ./.git:/opt/warehouse/src/.git:z
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏something about mounting the git directory feels weird to me. I'm guessing this has something to do with the "last updated" time in the blog engine. 
Would setting `:ro` instead of `:z` make sense here, so that we get a read-only version of the external git directory installed and thus prevent permission collisions from being made inside the container?

Reply:
Yeah, we're not currently using this feature and I agree this feels weird, so I'll remove it.